### PR TITLE
chore(updatecli): add a link in the PR description

### PR DESCRIPTION
**Pull request by smerle33**
more information in the PR

### CODE REVIEW

File: updatecli/updatecli.d/contributor.jenkins.io.expiry.yaml [hunk 1]
```diff
@@ -67,5 +67,6 @@ actions:
     scmid: default
     spec:
       title: Bump contributors.jenkins.io azurerm_storage_account_sas `expiry` date {{ source "nextExpiry" }}
+      description: paste this certicate output here (private repo) https://github.com/jenkins-infra/charts-secrets/blob/main/config/infra.ci.jenkins.io/jenkins-secrets.yaml variable CONTRIBUTORS_JENKINS_IO_FILESHARE_SAS_QUERYSTRING
```
Comment (dduportal) [inline updatecli/updatecli.d/contributor.jenkins.io.expiry.yaml:70]:
```suggestion
      description: paste this certicate output here (private repo) https://github.com/jenkins-infra/charts-secrets/blob/main/config/infra.ci.jenkins.io/jenkins-secrets.yaml (in the variable `CONTRIBUTORS_JENKINS_IO_FILESHARE_SAS_QUERYSTRING`)
```

**Review by dduportal — APPROVED** [review]
LGTM!

A non blocking suggestion (I'll open a PR if you don't take the suggestion in account)

File: updatecli/updatecli.d/contributor.jenkins.io.expiry.yaml (hunk 1 above)
Comment (lemeurherve) [reply updatecli/updatecli.d/contributor.jenkins.io.expiry.yaml:70]: I would remove the parenthesis from this suggestion.

Comment (lemeurherve) [reply updatecli/updatecli.d/contributor.jenkins.io.expiry.yaml:70]:
```suggestion
      description: Retrieve the `contributors_jenkins_io_sas_query_string` output to update the `CONTRIBUTORS_JENKINS_IO_FILESHARE_SAS_QUERYSTRING` variable value in https://github.com/jenkins-infra/charts-secrets/blob/main/config/infra.ci.jenkins.io/jenkins-secrets.yaml
```

File: updatecli/updatecli.d/contributor.jenkins.io.expiry.yaml [hunk 2]
```diff
@@ -67,5 +67,6 @@ actions:
     scmid: default
     spec:
       title: Bump contributors.jenkins.io azurerm_storage_account_sas `expiry` date {{ source "nextExpiry" }}
```
Comment (lemeurherve) [inline updatecli/updatecli.d/contributor.jenkins.io.expiry.yaml:69]:
```suggestion
      title: Increase contributors.jenkins.io SAS token `expiry` date to {{ source "nextExpiry" }}
```

**Review by dduportal — APPROVED** [review]

**Review by lemeurherve — APPROVED** [review]
🚢

**Pull request merged**